src/format_python.js

```diff
@@ -121,6 +121,21 @@ class PythonFormatter {
   filterComment(spec) {
     return spec.codeLang === this.lang;
   }
+
+  /**
+   * @param {Documentation.Member} member
+   */
+  memberSection(member) {
+    if (member.deprecated || member.discouraged)
+      return { key: 'd', title: 'Deprecated' };
+    if (member.kind === 'event')
+      return { key: 'c', title: 'Events' };
+    if (member.kind === 'property' || (member.kind === 'method' && !member.async && member.argsArray.length === 0))
```

Comment:
I'd instead introduce `renderSyncNoArgsMethodAsProperty(): boolean` method and put this logic into `generator.js`.